src/test/java/io/aiven/kafka/connect/s3/testutils/BucketAccessor.java

```diff
@@ -50,6 +54,29 @@ public final void createBucket() {
         s3.createBucket(bucketName);
     }
 
+    public final void removeBucket() {
+        final var chunk = s3.listObjects(bucketName)
```

Comment:
From S3 client docs:
```java
     * Because buckets can contain a virtually unlimited number of keys, the
     * complete results of a list query can be extremely large. To manage large
     * result sets, Amazon S3 uses pagination to split them into multiple
     * responses. Always check the
     * {@link ObjectListing#isTruncated()} method to see if the returned
     * listing is complete or if additional calls are needed to get
     * more results. Alternatively, use the
     * {@link AmazonS3Client#listNextBatchOfObjects(ObjectListing)} method as
     * an easy way to get the next page of object listings.
```
I think better use the `while` loop for removing files from the bucket. For `ListObjectsV2Request` you can set maxResult for the page

Reply:
This all make sence. But I know it is not large because it is just test helper, given that we delete bucket after each tests

Reply:
For testing purpose we can skip continuation token